### ISSUE TITLE
Bridges don't open when too close to a station

### ISSUE THREAD

**Issue by snouz** [rudegrass/cargo_ships#65]
While working on the mod, I found that bridges don't open at all if they're too close to a station. No visual indication, they just stay an obstacle.

![Imgur](https://i.imgur.com/4pNWdGf.png)

I checked that it wasn't one of my changes and created a new game with v0.0.59, created a very basic loop, and the same thing happens.

**Comment by squidy5** [comment]
How is the behavior when you place (chain) buoys before and behind the bridge entity?

**Comment by robot256** [comment]
Yes, I think the problem is that the bridge adds invisible signals, but requires additional signals before and after for the ship detection to work correctly.  In my last commit I tried to clarify this in the description.
